pwnagotchi/plugins/__init__.py

```diff
@@ -44,16 +44,19 @@ def load_from_file(filename):
     return plugin_name, instance
 
 
-def load_from_path(path, enabled=()):
+def load_from_path(path, enabled=list()):
```

Comment:
So this is (probably) not gonna do what you want here. If you call this function with no 'enabled', argument, python will use the *same* (mutable) list object every time you call this function. Check this out:

https://docs.python-guide.org/writing/gotchas/

and my simple example:
test.py:
```
def wtf(arg=list()):
    arg.append('a')
    return arg

print wtf()
print wtf()
```

```
jeremy@x1:~/pwnagotchi$ python test.py
['a']
['a', 'a']
```

Reply:
very good catch!